src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -86,6 +92,11 @@
 
 public class LibraryTab extends Tab {
```

Comment:
Can you add design-ideas as JavaDoc?

Reply:
Nope, at least not yet. Would be speculation. Right now its only to get some order into the chaos.
In fact, there are still other things I wanted to do here, but i stopped to keep this reviewable. But one future step would be to get the entry editor out of the library tab and into the frame, and to fix all the bindings stuff.

Reply:
Another thing would be to break this class into a more classes and fixing the mvvm pattern with library tab.